### ISSUE TITLE
Two BLOSC filters, Identifier Name both links to BLOSC(1)

### ISSUE THREAD

**Issue by mkitti** [HDFGroup/hdf5_plugins#112]
In https://github.com/HDFGroup/hdf5_plugins/blob/master/docs/RegisteredFilterPlugins.md there are two BLOSC filters described. One is actually BLOSC(1), 32001, and the other is BLOSC2, 32026. Currently the "Identifier Name" column links both BLOSCs to BLOSC(1).

Please disambiguate BLOSC(1) and BLOSC2.

**Comment by byrnHDF** [comment]
Yes already saw that and will fix when I add blosc2 support this week or next

**Comment by byrnHDF** [comment]
Updated and fixed
